src/vs/workbench/contrib/notebook/test/browser/cellOperations.test.ts

```diff
@@ -24,8 +24,10 @@ suite('CellOperations', () => {
 			],
 			async (editor, viewModel) => {
 				viewModel.updateSelectionsState({ kind: SelectionStateType.Index, focus: { start: 1, end: 2 }, selections: [{ start: 1, end: 2 }] });
-				await moveCellRange({ notebookEditor: editor, cell: viewModel.cellAt(1)! }, 'down');
+				const cell = viewModel.cellAt(1)!;
```

Comment:
I'm not a fan of non-null assertions as I feel like it will lead to bugs. Why can't we just `assert.ok(cell)` after calling this so we can be sure

Reply:
Good suggestion, `assert.ok` is neat.